src/Tizen.System.Feedback/Feedback/Feedback.cs

```diff
@@ -316,26 +258,15 @@ public bool IsSupportedPattern(FeedbackType type, String pattern)
         /// </example>
         public void Play(FeedbackType type, String pattern)
         {
-            bool found = false;
-            int i = 0;
             Interop.Feedback.FeedbackError res;
 
-            for (i = 0; i < Constants.NumberOfPattern; i++)
-            {
-                if (String.Compare(pattern, Pattern[i].PatternString) == 0)
-                {
-                    found = true;
-                    break;
-                }
-            }
-
-            if (!found)
+            if (!Pattern.ContainsKey(pattern))
```

Comment:
Using ContainsKey and Item operator at the same time is redundant, because we use ContainsKey inside Items operator.
I recommend using TryGetValue.
```C#
if (Pattern.TryGetValue(pattern, out int value))
{
    if (type == FeedbackType.All)
        res = (Interop.Feedback.FeedbackError)Interop.Feedback.Play(value);
    else
        res = (Interop.Feedback.FeedbackError)Interop.Feedback.PlayType((Interop.Feedback.FeedbackType)type, value);
}
else throw new ArgumentException("Invalid Arguments");
```

Reply:
Thank you for point this, TryGetValue() seems more efficient way to access element of Dictionary. :)